static/app/utils/fields/index.ts

```diff
@@ -1041,6 +1047,128 @@ export const DISCOVER_FIELDS = [
   SpanOpBreakdown.SpansUi,
 ];
 
-export const getFieldDefinition = (key: string): FieldDefinition | null => {
-  return FIELD_DEFINITIONS[key] ?? null;
+enum ReplayFieldKey {
+  BROWSER_NAME = 'browser.name',
+  BROWSER_VERSION = 'browser.version',
+  COUNT_ERRORS = 'countErrors',
+  COUNT_SEGMENTS = 'countSegments',
+  // COUNT_URLS = 'countUrls',
+  DEVICE_MODEL = 'device.model',
+  DURATION = 'duration',
+  // ERROR_IDS = 'errorIds',
+  // LONGEST_TRANSACTION = 'longestTransaction',
+  OS_NAME = 'os.name',
+  OS_VERSION = 'os.version',
+  RELEASES = 'releases',
+  // TRACE_IDS = 'traceIds',
```

Comment:
leaving a few things in here which could be good to support

Maybe we can throw them onto the list here: https://github.com/getsentry/sentry/pull/40691/files
cc @JoshFerge